logger.go

```diff
@@ -117,6 +154,27 @@ func (l *Logger) TimeFormat() string {
 	return l.options.TimeFormat
 }
 
+// Writer returns a io.Writer for the given level. It is generally combined with
+// the Logger.StdLogger method.
+func (l *Logger) Writer(level Level) io.Writer {
+	return &writer{
+		Logger: l.Logger,
+		Name:   zap.String("name", l.name),
+		Level:  level,
+	}
+}
+
+// StdLogger returns a *log.Logger with the specified writer. The writer can be
+// created with the Logger.Writer method.
+//  srv := &http.Server{
+//      Addr:     ":8080",
+//      Handler:  httplog.Middleware(logger, http.DefaultServeMux),
+//      ErrorLog: logger.StdLogger(logger.Writer(logging.ErrorLevel)),
+//  }
+func (l *Logger) StdLogger(w io.Writer) *log.Logger {
```

Comment:
Not sure if `StdLogger` has to exist on `Logger`; there's currently no relation to the `Logger` itself. 

Alternatively, it could take the logging level directly instead of the `io.Writer`, skipping the `logger.Writer` invocation. I don't know if you had plans for injecting different `io.Writer`s?

Reply:
Makes sense, I don't see the need of using a different writer, but we need the level.

Reply:
Fixed with 7df2edc